docs/protocol/terminology.mdx

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_position: 2
+title: "Terminology"
+---
+
+import Tooltip from '@site/src/components/Tooltip';
+
+# Terminology
+
+## Core Concepts
+
+**Channel**: A secure communication pathway between participants that locks funds in an on-chain smart contract while enabling off-chain state updates.
+
+**State**: A snapshot of the channel at a point in time, including fund allocations and application-specific data.
+
+**Participant**: An entity (identified by a wallet address) that is part of a channel. Typically includes a Creator (user) and a Clearnode or Clearnode Operator.
+
+**Clearnode**: A virtual ledger layer that provides a unified ledger (through Nitro RPC) and coordinates state channels (through Nitrolite), providing chain abstraction for developers and users.
```

Comment:
I think we should specify that Clearnode is a constantly running server, that provides such functionality. However, I am not sure of the best formulation.
Maybe something alike "Clearnode: a server implementing virtual ledger layer that provides ..."